### ISSUE TITLE
Improving insight into the robots behaviour.

### ISSUE THREAD

**Issue by SeaTechRC** [zauberzeug/field_friend#220]
Hello,

In order for a user to see why the robot has stopped the automation,
the reason should be kept and shown under the start/pause and stop button.
Otherwise, if the robot stops while being unattended, the user will not know what caused it.

Same would go for temporary stops (RTK fix lost, bumper pressed, etc.).

It could go even as far as showing the current state of the implement and the navigation, since both are state-based (or theoretically could be).
This could also be very nicely visualized with mermaid diagrams.

**Comment by rodja** [comment]
Yes, sounds great! We already have such a state representation and logging view for other robots.
